### PR TITLE
blogDetailPage#006

### PR DESCRIPTION
**Pull request by Salim-54**
#### What does this PR do?

- Shows the blog details
- Provides the comment form
- Shows comments

#### Description of Task to be completed? 

As a user, I should be able to see
- Blogs headings and titles
- the form to write my comment
- Comments
- Likes 

#### How should this be manually tested?

- Clone the repo

- checkout to branch ``blogDetailPage#006``

#### Any background context you want to provide?

- N/A

#### What are the relevant Pivotal Tracker/trello stories?

- [link to my pivotal tracker/tello/jira ... implemented story](https://trello.com/c/B7UED6tY/23-blogdetailpage006)

#### Screenshots (if appropriate)

Screenshot (256)
Screenshot (257)
Screenshot (255)

#### Questions:

- N/A

### CODE REVIEW

**Review by PrinceNiyonshuti — APPROVED** [review]
great work and keep it up

File: css/articles.css [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  --article-one: linear-gradient(rgba(188, 188, 182, 1),rgba(255, 255, 153, 1));
+  --article-two: linear-gradient(rgba(135, 141, 159, 1),rgba(215, 223, 255, 1));
+  --article-three: linear-gradient(rgba(29, 185, 176, 1),rgba(124, 245, 239, 1));
+  --article-four: linear-gradient(rgba(205, 156, 163, 1),rgba(255, 233, 236, 1));
+  --nav-bg:rgba(82, 93, 98, 1);
+  --detail-blog:rgba(136, 157, 168, 1);
+  --active-nav-link: rgba(160, 180, 255, 1);
+  --black-text: rgba(85, 92, 94, 1);
+  --white-text: #fff;
+  --light-grey-text:rgba(222, 224, 225, 1)
```
Comment (arthurjunior250) [inline css/articles.css:47]: try to reduce blank space

**Review by arthurjunior250 — COMMENTED** [review]
good job

File: css/articles.css [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+
+; }
```
Comment (arthurjunior250) [inline css/articles.css:50]: try to reduce blank space

**Comment by RobertKaySr** [comment]
This is good, keep it up, great work

**Pull request merged**